openstates/co/utils/email.py

```diff
@@ -0,0 +1,4 @@
+def mailto_to_email(mailto_url):
+    """Return a single email extracted from 'mailto:name@domain.com'"""
+    email = mailto_url.lstrip('mailto:')
```

Comment:
TIL. In going to fix this, saw they updated the main page to include an XLSX download, and expanded the table to include email. I'm going to update the scraper to pull the email from there.  Do we prefer Excel to HTML?